websites/N/Neon/dist/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Neon",
+    "author": {
+      "name": "Kety The Folf",
+      "id": "263443630767734784"
+    },
+    "contributors": [
+        {
+          "name": "SlowLife",
+          "id": "374905512661221377"
+        }
+      ],
```

Comment:
No contributors on release.

Reply:
Nah SlowLife actually helped Kety a lot, I saw in #presence-dev, its okay this time.